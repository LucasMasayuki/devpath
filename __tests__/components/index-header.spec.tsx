import React from 'react'
import renderer from 'react-test-renderer'
import IndexHeader from '@/components/index-header'

describe('IndexHeader', () => {
  it('should render index header', () => {
    const component = renderer.create(<IndexHeader />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
