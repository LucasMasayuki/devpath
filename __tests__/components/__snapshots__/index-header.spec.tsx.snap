// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IndexHeader should render index header 1`] = `
<div
  className="css-113utp"
>
  <h1
    className="chakra-heading css-0"
  >
    DEVMON
  </h1>
  <code
    className="chakra-code css-kbxewa"
  >
    // Developer in eternal evolution
  </code>
</div>
`;
